controls/utilities.py

```diff
@@ -78,3 +78,20 @@ def oscalize_catalog_key(catalogkey):
         catalogkey = split_key_list[0] + "-800-" + split_key_list[1]
 
     return catalogkey
+
+def get_control_statement_part(control_stmnt_id):
```

Comment:
I assume we will call it on that style control identifier. I would think this parser would still work though, because assuming the statement with a part would look like "3.2.1_smt.b", the parser in either case is just looking for the part after "_smt.", which is the same in both situations. Right?